instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LoggerE2ETests.kt

```diff
@@ -501,6 +501,7 @@ class LoggerE2ETests {
         val CUSTOM_JSON_OBJECT_ATTRIBUTE: JsonObject = JsonObject().apply {
             addProperty(CUSTOM_JSON_OBJECT_PROPERTY, CUSTOM_INT_ATTRIBUTE)
         }
-        val CUSTOM_DATE_ATTRIBUTE: Date = SimpleDateFormat("yyyy-MM-dd").parse("2021-01-01")!!
+        val CUSTOM_DATE_ATTRIBUTE: Date = SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS")
```

Comment:
all the values of `test_special_date_attribute` I found still point to the start of the hour with 0 minutes and 0 seconds, so probably it is just a timezone issue (local machine vs Bitrise instance)? Probably we just need to specify timezone in this `SimpleDateFormat`?

Reply:
Yes when dealing with dates, specifying timezone is required as we don't know how the underlying machine's locale is configured

Reply:
That's a very good point, will add the `UTC` timezone there just to make sure.